src/main.rs

```diff
@@ -26,9 +26,10 @@ async fn main() {
           .get_matches();
 
      let pub_address = get_pub_address(matches.value_of("seed")).await;
-     let network: &str = matches.value_of("network").unwrap_or("");
+     let network: &str = matches.value_of("network").unwrap_or("substrate");
 
-     let address: String = pub_address.to_ss58check_with_version(get_network_format(network));
+     let address: String =
+          pub_address.to_ss58check_with_version(network.parse().unwrap_or(Default::default()));
```

Comment:
For this particular case it's probably better to use the `_or_else` variant which is lazy, right now the `Default::default` is always executed even when the parsing is successful and might involve non trivial amount of work(it actually does, its implementation executes a Mutex lock)